.releaserc

```diff
@@ -0,0 +1,6 @@
+{
+  branches: [
+    {name: 'master'},
```

Comment:
If we don't want to run the release process on this branch [as per here](https://github.com/teamsnap/teamsnap-ui/pull/940/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R5), should we remove this line?

Reply:
You are totally correct. good catch!